### DISCUSSION TITLE
Import Dependent Libraries

### DISCUSSION THREAD

**Discussion by LootDust** [tyron12233/CodeAssist#456] [Q&A]
As title, i dont know how to add Libraries. The build.gradle says that use Library Manager but in the Library Manager there is only a button called "Add libraries from build.gradle" which makes me confused.

**Comment by TheRandomCrafter83** [comment]
add the libraries to the gradle file like normal, then use that button to import from the gradle file. at this point in time, automatic processing of the gradle file isnt possible and needs to be done via this button.

**Reply by TrevoDng** [reply]
Hi, am also a new user, can you please just make an example of adding RecyclerView library because I really don't know how t do this and there is also no tutorial for adding dependencies.

**Reply by TrevoDng** [reply]
Where i find the library manager

**Reply by TheRandomCrafter83** [reply]
in the menus top right

**Reply by TrevoDng** [reply]
Thank you very much I got it now.

**Comment by Gabskit** [comment]
I think library import in code assist don't work
When i try import a library (example: patternlockview by Aritraroy), appears "unable to retrieve POM"

**Reply by Gabskit** [reply]
Finally i imported a Library 
But when i try to compile a apk this error appears 
![Screenshot_2025-05-26-15-02-27-633_com tyron code](https://github.com/user-attachments/assets/424f2a1d-cff1-4a61-89ab-fb260f4bf8ef)
I didn't even use something of the Library 
Abd finding in the mentioned files, looks like a conflict in default librarys
Please help me

**Comment by TheRandomCrafter83** [comment]
CodeAssist is no longer being developed. It is recommended to switch to Android Studio on the computer.Carl ***@***.***
-------- Original message --------From: Gabskit ***@***.***> Date: 5/26/25  5:04 PM  (GMT-05:00) To: tyron12233/CodeAssist ***@***.***> Cc: Carl Williams ***@***.***>, Comment ***@***.***> Subject: Re: [tyron12233/CodeAssist] Import Dependent Libraries (Discussion #456) 
Finally i imported a Library
But when i try to compile a apk this error appears
Screenshot_2025-05-26-15-02-27-633_com.tyron.code.jpg (view on web)
I didn't even use something of the Library
Abd finding in the mentioned files, looks like a conflict in default librarys
Please help me

**Reply by Gabskit** [reply]
Oh no
